packages/donations-app/app/components/global/storyCard.html

```diff
@@ -1,7 +1,7 @@
 <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 story-card__container">
   <div class="story-card">
     <!-- story card image -->
-    <img class="story-card__image" src="{{ story.img }}">
+    <img class="story-card__image lazy" data-src="{{ story.img }}">
```

Comment:
Shouldn't we add data-src in other places? Is data-src compatible with ng-src that we use e.g. in the project splash?

Reply:
Discussed on call